app/assets/stylesheets/custom.scss

```diff
@@ -84,10 +84,11 @@
   color: #ee5034
 }
 
-.numeric {
+.numeric,
+[data-quantity] {
```

Comment:
Is there a reason this is using a data attribute and not a CSS class?

Reply:
Updated to use a class.